components/Dashboard/Post/PostComments.tsx

```diff
@@ -71,7 +71,12 @@ const PostComments: React.FC<PostCommentsProps> = ({
               onChange={(e) => setPostCommentBody(e.target.value)}
               disabled={loading}
             />
-            <Button type="submit" disabled={loading} className="submit-btn">
+            <Button
+              type="submit"
```

Comment:
After submitting, if we call `refetch` then I think the comment should appear after submitting (right now it looks like you have to refresh the page). We'd just need to pass down `refetch` from `pages/post/[id]/index.tsx` to `<PostComments />`. In case you want to do that, you can see an example in `components/Modals/WelcomeModal/WelcomeModalBody.tsx` for how to make TypeScript happy for the `refetch` prop

Reply:
This is actually already happening 😄 Those `onNewComment`, `onUpdateComment`, and `onDeleteComment` props are actually all just `refetch` calls being passed down 🙂 There's no refresh necessary to see new comments

Reply:
awesome!! even better 😄 i couldn't get it to work locally, but I think that was something weird with my local state

Reply:
Weehoo! 😄 🙌🏼